src/commands/install.yml

```diff
@@ -1,20 +1,20 @@
 description: >
-  Install MATLAB on a Linux machine executor. Currently, this command is available only for public
-  projects and does not include transformation products, such as MATLAB Coder and MATLAB Compiler.
+  Install a specified version of MATLAB. If you use this orb to install transformation products, such
+  as MATLAB Coder™ and MATLAB Compiler™, the action does not automatically license such products for you.
 
 parameters:
   products:
     description: >
       Products to install in addition to MATLAB, specified as a list of product names separated by
       spaces. You can specify products to set up most MathWorks products and support packages. The
-      action uses MATLAB Package Manager (mpm) for installing products. For a list of supported
```

Comment:
Should mpm use a different font, such as `mpm`?

Reply:
I don't think we have access to different fonts, but it does seem like some other orbs are using `cli-name` syntax anyways. See \`dockerlint\` on this page for an example:
https://circleci.com/developer/orbs/orb/circleci/docker

Reply:
Yes, that's what I was thinking: using backticks to differentiate between text and code.